websites/A/AnimesOnline/iframe.ts

```diff
@@ -0,0 +1,18 @@
+const iframe = new iFrame();
+
+iframe.on("UpdateData", async () => {
+  let video: HTMLVideoElement;
+
+  if (document.querySelector(".jw-video") !== null)
+    video = document.querySelector(".jw-video");
+  else if (document.querySelector(".html5-video-container") !== null)
+    video = document.querySelector(".html5-video-container > video");
```

Comment:
```suggestion
  const video: HTMLVideoElement = document.querySelector(".jw-video") || document.querySelector(".html5-video-container");
```
This should work but test if it actually does!